kong/plugins/rate-limiting/migrations/001_14_to_15.lua

```diff
@@ -0,0 +1,16 @@
+return {
+  postgres = {
+    up = [[
+      ALTER TABLE IF EXISTS ONLY "ratelimiting_metrics"
+        ALTER "period_date" TYPE TIMESTAMP WITH TIME ZONE USING "period_date" AT TIME ZONE 'UTC';
+    ]],
+  },
+
+  cassandra = {
+    up = [[
+      BEGIN BATCH
+        -- nothing to migrate
+      APPLY BATCH;
+    ]],
```

Comment:
It should be possible (and preferable) to specify an empty string for these NOP migrations; have you tried?

Reply:
I tried, but it gave an error. We can fix why it errored, or keep this.

Reply:
Could we fix the error then? Specifying the migration is required for the sake of explicitness as per [the migrations schema](https://github.com/Kong/kong/blob/next/kong/db/schema/others/migrations.lua#L11-L12), but the connectors should just NOP on empty migrations

Reply:
@thibaultcha I added `len_min = 0` to migrations schema.